src/EnergyPlus/PoweredInductionUnits.cc

```diff
@@ -1608,6 +1593,9 @@ namespace PoweredInductionUnits {
 		QZnReq = ZoneSysEnergyDemand( ZoneNum ).RemainingOutputRequired;
 		QToHeatSetPt = ZoneSysEnergyDemand( ZoneNum ).RemainingOutputReqToHeatSP;
 		CpAirZn = PsyCpAirFnWTdb( Node( ZoneNode ).HumRat, Node( ZoneNode ).Temp );
+		// Initialize local fan flags to global system flags
+		PIUTurnFansOn = ( DataHVACGlobals::TurnFansOn || DataHVACGlobals::TurnZoneFansOnlyOn );
+		PIUTurnFansOff = DataHVACGlobals::TurnFansOff;
```

Comment:
Oh, that's it, these two optional parameters are assumed to get used as a pair, in [Fans::SimulateFanComponents](https://github.com/NREL/EnergyPlus/blob/8971859c03f8c5d8e350d46877364a5a6c60d1e4/src/EnergyPlus/Fans.cc#L271).
`		if ( present( ZoneCompTurnFansOn ) && present( ZoneCompTurnFansOff ) ) {`